modules/ossm-rn-new-features.adoc

```diff
@@ -11,6 +11,19 @@ Reason – If known, include why has the enhancement been implemented (use case,
 Result – If changed, describe the current user experience
 ////
 
+== New features {ProductName} 1.0
+
+This release of {ProductName} adds support for Istio 1.1.11, Jaeger 1.13.1, Kiali 1.0.5, and the 3scale Istio Adapter 1.0.
+
+Other notable changes in this release include the following:
+
+* The Kubernetes Container Network Interface (CNI) is always on.
```

Comment:
This needs to say "Istio CNI plugin", as CNI is always present (and "enabled") in every Kubernetes cluster. What the service mesh actually installs is just a CNI _plugin_ that the CNI support inside Kubernetes calls out to when starting up a pod.

Reply:
Fixed.